src/App.js

```diff
@@ -1,23 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Link, Route, Routes } from 'react-router-dom';
+import Greeting from './components/Greeting';
+import Home from './components/Home';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Link to="/greeting">View Greetings</Link>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/greeting" element={<Greeting />} />
+      </Routes>
     </div>
```

Comment:
- [ ] Kindly look into your frontend app. It displays nothing on the browser and the console contains a bunch of errors (as displayed in the image). Kindly look into that and fix the errors.

- [ ] I was unable to check if the greetings are actually fetched from the API due to these errors. Kindly ensure that works also. If you have cors-policy issue you can resolve it using [the `rack-cors` gem](https://github.com/cyu/rack-cors)

  ![image](https://user-images.githubusercontent.com/54848363/231260533-348db42d-69ed-41b0-8483-4270a7d2cc7b.png)